examples/todo/src/main.rs

```diff
@@ -78,8 +78,14 @@ fn index(msg: Option<FlashMessage>, conn: DbConn) -> Template {
     })
 }
 
-fn rocket() -> (Rocket, Option<DbConn>) {
-    let rocket = rocket::ignite()
+fn rocket(override_config: Option<Config>) -> (Rocket, Option<DbConn>) {
+    let rocket = if let Some(config) = override_config {
```

Comment:
Can this be `override_config.map(rocket::custom).unwrap_or_else(rocket::ignite)`?